ci/lib.sh

```diff
@@ -36,6 +36,9 @@ os() {
 
 arch() {
   case "$(uname -m)" in
+  armv7l)
+    echo arm64
+    ;;
```

Comment:
Wouldn't this break since armv7l is 32 bit? ([source](https://askubuntu.com/a/1042272))

Reply:
Let me answer the 2 questions in one:
arm7l can and should run the upcoming 64bit raspbian (hopefully bullseye, run nicely but as a alfa version), but right now the official version is buster and that is only 32bit, so I will move it.

I do this, because I want to contribute to code-server. I am at the moment very active in homeassistant also an apache member and have been programming since 1976 (professionally from 1983). My setup is an ipad with a raspberry pi 4 (with a nvm-e attached).

I have been trying to build a version, but have run into several problems, which I am solving one by one. If you have a better way of getting a development environment working on my raspberry pi 4 (buster), I am more than happy to use that.

Being a network expert (I am NOT a UI gui) I have been tracing the communication between the UI and the backend, and I think I know why I see instabilities, including why reconnect can lead to an error that a file cannot be saved.

So in short, I just want to be helpful :-)

Reply:
> arm7l can and should run the upcoming 64bit raspbian (hopefully bullseye, run nicely but as a alfa version), but right now the official version is buster and that is only 32bit, so I will move it.

Ah, gotcha! That makes sense. 

> If you have a better way of getting a development environment working on my raspberry pi 4

Oh man. That is a great question! I have a Raspberry Pi 4 but haven't tried developing code-server on it. I do also use an iPad occasionally for writing code, but I host code-server with Railway.

One idea that comes to mind is us creating a Railway template specifically for developing code-server. Then we wouldn't need to worry about issues with the Pi and you could develop from anywhere (including the iPad). Would that something you're interested in? 

> I do this, because I want to contribute to code-server
> So in short, I just want to be helpful :-)

Honestly, we can't tell you how much we appreciate that!

Reply:
I have an iMac, which I share with my wife, which is why I moved my development (apart from a big C++ project) to the Raspberry. My iPad is the center of all my activities, with a mouse and a external keyboard.

My idea from running code-server on a raspberry, is due to my involvement in HomeAssistant (a pure python project), but I can do fine with having a code-server in the cloud. At least that could get me up and running fast.

Reply:
Awesome! Okay, let me add it to my TODO list and see if I can put something together for you. It should be before the end of the week.

Reply:
@janiversen we took a look into it and unfortunately I don't think the Railway env will work 😢 @bpmct and I were looking into it and found some system requirements for VS Code that won't work on Railway's free plan. More details [here](https://github.com/bpmct/develop-code-server/issues/8#issuecomment-863411167).

> I will try to build code-server on the Mac, since I can easily cross compile

Maybe this will be the best solution

Reply:
Thank you for jumping in and contributing (even though your ran into roadblocks due to some of the limitations with our workflow and docs). 

> Thanks for merging my first (of hopefully many) commits.

Seriously, thank you for contributing! We have more work then bandwidth and every bit helps a ton! Hopefully looking forward to more commits!

Reply:
Speaking of bandwidth, I like to do the “dummy” tasks like doing intelligent replace of x to y+, because it is my way of getting into the code. So if any if you need that done, forward it to me and concentrate on the difficult issues.

Do we have a chat forum like discord ? it is a lot easier for discussions.

Reply:
> Speaking of bandwidth, I like to do the “dummy” tasks like doing intelligent replace of x to y+, because it is my way of getting into the cod

That's very kind of you to offer! I'm trying to think. I've added a bunch to the [Roadmap](https://github.com/cdr/code-server/projects/4). I think you should be able to see the Notes in there. Most of it is related to testing. Currently priority for me at least is to improve our code-coverage and get it to 100%. That might be a good place to start?

 > Do we have a chat forum like discord? 
 
We do have a Community Slack. https://community.coder.com/

Though if I'm being honest, I don't check it too much personally (not sure about the other maintainers) but it just becomes too much to deal with. We all check GitHub multiple times throughout the day though so a GitHub Discussion could work 🤔 If we find that too slow or doesn't work, happy to try something else with you!